lib/data-helper.js

```diff
@@ -88,6 +88,39 @@ export function getAccountConfig( account ) {
 	return localConfig[ account ];
 }
 
+export function getAllAccountsWithFeatures( features = [] ) {
+	let allAccounts = [];
+
+	if ( config.has( 'accounts' ) ) {
+		allAccounts = config.get( 'accounts' );
+	} else if ( process.env.ACCOUNT_INFO ) {
+		allAccounts = JSON.parse( process.env.ACCOUNT_INFO );
+	}
+
+	return allAccounts.filter( function( account ) {
+		// return accounts that have all the features requested
+		return difference( account.features || [], features ).length === 0;
```

Comment:
Is that what you wanted here?
```js
_.difference(['passwordless', '2fa'], ['passwordless']); // => [ '2fa' ]
```

shouldn't `features` be a subset of `account.feature` ?

Reply:
Nice catch!, thx!